app/lib/dfe/reference_data/tweaked_reference_list.rb

```diff
@@ -0,0 +1,66 @@
+module DfE
+  module ReferenceData
+    ##
+    #
+    # A +TweakedReferenceList+ is a wrapper around a +ReferenceList+ that applies
+    # some local "tweaks" - overrides to the underlying list that either add new
+    # records, add new fields to existing records, overwrite existing fields in
+    # existing records, or hide some records.
+    class TweakedReferenceList < ReferenceList
```

Comment:
I got a bit confused about the names of some of these collections. Is a `TweakedReferenceList` basically a `ReferenceList` with a set of overridden keys?

Reply:
Maybe we could leave it out for now? I can see where it's used.